### ISSUE TITLE
Display path details without elevation data

### ISSUE THREAD

**Issue by otbutz** [graphhopper/graphhopper#2267]
It would be nice to have the graph also available for debugging purposes even if no elevation data is present.

![image](https://user-images.githubusercontent.com/17603532/111612467-c6565180-87dd-11eb-9440-5256c2ce07bd.png)

e.g display a flat 0m graph with color gradient for `elevation=false`, `debug=true` and `details`

**Comment by easbar** [comment]
Yes. I thought there is a similar issue already but I cannot find it. Not sure how much effort we should put into the current GH maps though, because the rewrite is on its way: https://github.com/graphhopper/graphhopper-maps

**Comment by easbar** [comment]
> e.g display a flat 0m graph with color gradient for elevation=false + debug=true + details=

Even without `elevation=false&debug=true` the details should be shown, shouldn't they? The only thing that might be a bit irritating is that we show elevation on the y-axis, so maybe when there is no elevation data we should change the y-axis labels as well?
